scripts/configure/k8s.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -ex
+mkdir -p $(pwd)/tmp
+
+export BRIDGE_K8S_AUTH="bearer-token"
+export BRIDGE_USER_AUTH="disabled"
+export BRIDGE_K8S_MODE="off-cluster"
+export BRIDGE_K8S_MODE_OFF_CLUSTER_SKIP_VERIFY_TLS=true
+export BRIDGE_USER_SETTINGS_LOCATION="localstorage"
+
+export BRIDGE_K8S_MODE_OFF_CLUSTER_ENDPOINT=$(kubectl config view -o jsonpath="{.clusters[?(@.name==\"$(kubectl config current-context)\")].cluster.server}")
```

Comment:
Can we first check if the env var is present? i.e.
`K8S_AUTH_BEARER_TOKEN=${BRIDGE_K8S_AUTH_BEARER_TOKEN:="abcdef.0123456789abcdef"}`
In my flow I install the cluster on a different machine and need to overwrite at least endpoint:
`cluster.server`  address is local here i.e. `https://127.0.0.1:6443`

Reply:
added fall back for cluster endpoint and for cluster token